sh/install.sh

```diff
@@ -153,46 +170,87 @@ function install_wavecli() {
 
 }
 
+function detect_python(){
+    PYTHON_PATH=$(which python)
+
+    if [ -z "$PYTHON_PATH" ]; then
+        PYTHON_PATH=$(which python3)
+    fi
+
+    if [ -z "$PYTHON_PATH" ]; then
+        PYTHON_PATH=$(which python2)
+    fi
+
+    if [ -z "$PYTHON_PATH" ]; then
+        echo ""
+    else
+        echo "$PYTHON_PATH"
+    fi
+
+}
+
+function detect_pip(){
+    PIP_PATH=$(which pip)
```

Comment:
It might be safer to use `pythonX.Y -m pip` with the specific version of python we detected, as this logic might put us into python3/pip2 or python2/pip3 mismatch

Reply:
we were using the python to just install the pip. I have updated the logic to first check for pip and if pip is installed then no need to check for python, otherwise check for Python and install if not found and use the same python for installing pip.